src/nuclide.cpp

```diff
@@ -169,11 +169,10 @@ Nuclide::Nuclide(hid_t group, const std::vector<double>& temperature)
   // Sort temperatures to read
   std::sort(temps_to_read.begin(), temps_to_read.end());
 
-  double T_min_read = *std::min_element(temps_to_read.cbegin(), temps_to_read.cend());
-  double T_max_read = *std::max_element(temps_to_read.cbegin(), temps_to_read.cend());
-
-  data::temperature_min = std::max(data::temperature_min, T_min_read);
-  data::temperature_max = std::min(data::temperature_max, T_max_read);
+  data::temperature_min =
+    std::min(data::temperature_min, static_cast<double>(temps_to_read.front()));
```

Comment:
Will this always choose 0 as that's what `temperature_min` is initialized to on line 35?

Reply:
Ah, good catch. So we need to initialize temperature_min to INFTY instead.

Reply:
If it's initialized to INFTY, can we have the case where minimum temperature ends up being INFTY while the maximum temperature ends up being zero?

Reply:
It should be impossible, so long as any nuclide gets loaded. If a nuclide does not get loaded, they will stay put.

Reply:
In the old method, if no nuclides are loaded then the min is 0 and the max is INFTY, which makes enough sense. In the new method, we end up with min = INFTY and max = 0, which does not make sense. But, perhaps if no nuclides are loaded then we don't care about these fields? I can't think of a case where we would.

Reply:
Yeah, definitely agree that a void-only case will have no meaningful interpretation of temperature.